docs/source/getting_started/dataset_creation/dataset_creation_aws_bucket.rst

```diff
@@ -87,6 +87,64 @@ Let us assume your bucket is called `datalake`. And let us assume the folder you
 
         Get security credentials (access key id, secret access key) from AWS
 
+
+
+
+**S3 IAM Delegated Access**
+
+To access your data in your S3 bucket on AWS, Lightly `can assume a role <https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html>`_ in your account which has the necessary permissions to access your data.
+This is `considered best practice <https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#delegate-using-roles>`_ by AWS.
```

Comment:
Should't this come before the `Setting up IAM` section because it is best practice?